nearcore/src/config.rs

```diff
@@ -1161,12 +1162,33 @@ async fn download_file_impl(
     let https_connector = hyper_tls::HttpsConnector::new();
     let client = hyper::Client::builder().build::<_, hyper::Body>(https_connector);
     let mut resp = client.get(uri).await.map_err(FileDownloadError::HttpError)?;
+    let bar = match resp.size_hint().upper() {
```

Comment:
I’d use `if let` but `match` also works I suppose.

Reply:
Yeah, I guess that is the more canonical way of doing that